cheatsheets/crypto.md

```diff
@@ -47,3 +47,31 @@
 ```
 0123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345
 ```
+
+**Length extension attack**
+
+In cryptography and computer security, a length extension attack is a type of attack where an attacker can use `Hash(message1)` and the length of message1 to calculate `Hash(message1 ∥ message2)` for an attacker-controlled message2.
+
+TL;DR: given a hash that is composed of a string with an unknown prefix, an attacker can append to the string and produce a new hash that still has the unknown prefix.
```

Comment:
We should probably try to stay away from abbreviations that have not been defined earlier. Maybe add "In summary".